lib/id/index.js

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+const didKeyDriver = require('@digitalbazaar/did-method-key').driver();
+
+/**
+ * Generates a key pair. If no seed is given, a random one is generated.
+ *
+ * @param {object} options - Options hashmap.
+ * @param {{encoded: string, bytes: Uint8Array}} options.secretKeySeed -
+ *   32 byte random seed value, multihash identified, and multibase-encoded.
+ * @param {string} options.method - DID Method.
+ *
+ * @returns {Promise<{keyPair: object, secretKeySeed: string}>} - Resolves
+ *   with a key pair, and optionally a text-encoded secret seed (if none was
+ *   provided).
+ */
+export async function create({secretKeySeed, method} = {}) {
+  let didDocument;
+  switch(method) {
+    case 'key':
+      ({didDocument} = await didKeyDriver.generate({
+        seed: secretKeySeed.bytes
+      }));
+      break;
```

Comment:
I would think did-io could handle this for us -- or a Map could (instead of a switch statement). But not going to hold this up over either.

Reply:
@dlongley - so, we ended up not having a general `didIo.generate()` function, across the methods.
Partly it's because the various method drivers require different parameters.
We could add it, though.
(Also, see the related discussion on https://github.com/veres-one/did-veres-one/issues/74, about adding a `seed` param to Veres One `.generate()`.)

Reply:
Right, so, we ended up not having a general-purpose `didIo.generate()` function -- partly because the various generate methods had such different parameters.
We could add one though.

See also the discussion over at https://github.com/veres-one/did-veres-one/issues/74 (which is a proposal to add a `seed` parameter to Veres One's `.generate()`)

Reply:
@dlongley - I added `.generate()` to did-io over at https://github.com/digitalbazaar/did-io/pull/60,
and updated the code to use it.

See if you like that better.